src/com/simpleclients/connection/Connection.java

```diff
@@ -0,0 +1,83 @@
+package com.simpleclients.connection;
```

Comment:
not a super big problem, but the top level package should be the inverse of a domain name you own. Lacking that something unique would suffice.

so either com.<domain>.simpleclients or something unique that isn't a valid domain would work as well.

then again, it's not super important especially when you can still change it later

Reply:
I was thinking that when reviewing today. I was going to dig through domains to see if I actually own one currently lol